### PR TITLE
common/[abc]*: add link

### PR DESCRIPTION
**Pull request by marchersimon**
For #5510
Maybe related to #3016

### CODE REVIEW

File: pages.ko/common/badblocks.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > 불량 블록이 있는지 장치를 검사하십시오.
 > 불량 블록을 사용하면 파티션 테이블을 포함하여 디스크의 모든 데이터를 지우는 등의 파괴적인 작업이 발생할 수 있습니다.
+> 더 많은 정보: <https://man.archlinux.org/man/ar.1>.
```
Comment (navarroaxel) [inline pages.ko/common/badblocks.md:5]: `ar` ⸮

File: pages.ko/common/crontab.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 > 현재 사용자의 시간 간격으로 cron작업이 실행되도록 스케줄.
 > 작업 정의 형식: "(분) (시) (날짜) (달) (요일) 실행 할 명령".
+> 더 많은 정보: <https://man.archlinux.org/man/core/cronie/crontab.1>.
```
Comment (navarroaxel) [inline pages.ko/common/crontab.md:5]:
```suggestion
> 더 많은 정보: <https://man.archlinux.org/man/crontab.1>.
```

File: pages.zh/common/apropos.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # apropos
 
 > 在 manpages 中搜索，例如查找一个新命令.
+> 更多信息： <https://man.archlinux.org/man/core/man-db/apropos.1>.
```
Comment (navarroaxel) [inline pages.zh/common/apropos.md:4]:
```suggestion
> 更多信息： <https://man.archlinux.org/man/apropos.1>.
```

File: pages/common/apropos.md [hunk 4]
```diff
@@ -1,15 +1,16 @@
 # apropos
 
-> Search in manpages, for example to find a new command.
+> Search the manual pages for names and descriptions.
+> More information: <https://man.archlinux.org/man/core/man-db/apropos.1>.
```
Comment (navarroaxel) [inline pages/common/apropos.md:4]:
```suggestion
> More information: <https://man.archlinux.org/man/apropos.1>.
```

File: pages/common/crontab.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 > Schedule cron jobs to run on a time interval for the current user.
 > Job definition format: "(min) (hour) (day_of_month) (month) (day_of_week) command_to_execute".
+> More information: <https://man.archlinux.org/man/core/cronie/crontab.1>.
```
Comment (navarroaxel) [inline pages/common/crontab.md:5]:
```suggestion
> More information: <https://man.archlinux.org/man/crontab.1>.
```

File: pages.ko/common/badblocks.md (hunk 1 above)
Comment (marchersimon) [reply pages.ko/common/badblocks.md:5]: Whoops! probably a wrong copy-paste

File: pages.it/common/apg.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 # apg
 
 > Crea password randomiche arbitrariamente complesse.
+> Maggiori informazioni: <https://linux.die.net/man/1/apg>.
```
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages.it/common/apg.md:4]: Oh, yeah, I wanted to comment on that. I couldn't find anything else, so I just took that for now. Maybe we can find something better.

File: pages/common/apropos.md [hunk 7]
```diff
@@ -1,15 +1,16 @@
...
 
 `apropos {{regular_expression}}`
 
-- Search without restricting output to terminal width:
+- Search without restricting the output to the terminal width:
 
 `apropos -l {{regular_expression}}`
 
-- Search for pages that only contain all of the expressions given (AND search):
+- Search for pages only that contain all of the expressions given (AND search):
```
Comment (navarroaxel) [inline pages/common/apropos.md:14]: Not sure about this grammar @bl-ue

File: pages.it/common/apg.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.it/common/apg.md:4]: Here's some:

- https://manpages.debian.org/latest/apg/apg.1.html
- http://manpages.ubuntu.com/manpages/latest/man1/apg.1.html

I like the debian one better.

[the remark this answers is not in the document]
Comment (marchersimon) [reply pages.it/common/apg.md:4]: Looks good, thanks. I'm wondering why I didn't find that.

File: pages.it/common/apg.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 # apg
 
 > Crea password randomiche arbitrariamente complesse.
+> Maggiori informazioni: <https://manpages.debian.org/latest/apg/apg.1.html>.
```
Comment (navarroaxel) [inline pages.it/common/apg.md:4]:
```suggestion
> Maggiori informazioni: <https://manpages.debian.org/latest/apg/apg.1>.
```

File: pages.ko/common/apg.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 # apg
 
 > 임의로 복잡한 랜덤 암호를 만듭니다.
+> 더 많은 정보: <https://manpages.debian.org/latest/apg/apg.1.html>.
```
Comment (navarroaxel) [inline pages.ko/common/apg.md:4]:
```suggestion
> 더 많은 정보: <https://manpages.debian.org/latest/apg/apg.1>.
```

File: pages.pt_BR/common/apg.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 # apg
 
 > Criar senhas aleatórias arbitrariamente complexas.
+> Mais informações: <https://manpages.debian.org/latest/apg/apg.1.html>.
```
Comment (navarroaxel) [inline pages.pt_BR/common/apg.md:4]:
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apg/apg.1>.
```

File: pages/common/apg.md [hunk 11]
```diff
@@ -1,6 +1,7 @@
 # apg
 
 > Creates arbitrarily complex random passwords.
+> More information: <https://manpages.debian.org/latest/apg/apg.1.html>.
```
Comment (navarroaxel) [inline pages/common/apg.md:4]:
```suggestion
> More information: <https://manpages.debian.org/latest/apg/apg.1>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/apropos.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/apropos.md:14]: Yeah, the original sounds better (`that only`).  How about this:
```suggestion
- Search for pages that contain all of the expressions given:
```

File: pages.it/common/ar.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
 # ar
 
 > Crea, modifica ed estrai da archivi (.a, .so, .o).
```
Comment (bl-ue) [inline pages.it/common/ar.md:3]:
```suggestion
> Crea, modifica ed estrai da archivi (`.a`, `.so`, `.o`).
```

File: pages.ko/common/ar.md [hunk 13]
```diff
@@ -1,6 +1,7 @@
 # ar
 
 > 아카이브로부터 생성, 수정, 추출 (.a, .so, .o).
```
Comment (bl-ue) [inline pages.ko/common/ar.md:3]:
```suggestion
> 아카이브로부터 생성, 수정, 추출 (`.a`, `.so`, `.o`).
```

File: pages.zh/common/ar.md [hunk 14]
```diff
@@ -1,6 +1,7 @@
 # ar
 
 > 创建，修改，提取库文件 (.a, .so, .o).
```
Comment (bl-ue) [inline pages.zh/common/ar.md:3]:
```suggestion
> 创建，修改，提取库文件 (`.a`, `.so`, `.o`).
```

File: pages/common/ar.md [hunk 15]
```diff
@@ -1,23 +1,24 @@
 # ar
 
 > Create, modify, and extract from archives (.a, .so, .o).
```
Comment (bl-ue) [inline pages/common/ar.md:3]:
```suggestion
> Create, modify, and extract from archives (`.a`, `.so`, `.o`).
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: pages.it/common/column.md [hunk 16]
```diff
@@ -1,20 +1,21 @@
...
-`printf "intestazione1 intestazione2\nbar foo\n" | column -s{{,}}`
+`printf "intestazione1 intestazione2\nbar foo\n" | column --separator {{,}}`
 
 - Separa colonne ed allinea automaticamente in un formato tabulare:
 
-`printf "intestazione1 intestazione2\nbar foo\n" | column -t`
+`printf "intestazione1 intestazione2\nbar foo\n" | column --table`
 
-- Riempi le colonne prima delle righe:
+- Riempi le righe prima delle colonne:
```
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.it/common/column.md:19]: Btw @marchersimon how did you discover this?

[the remark this answers is not in the document]
Comment (marchersimon) [reply pages.it/common/column.md:19]: When I searched for the long arguments in the english page, I found, that the description of the last example was incorrect (column and row was swapped) and with a little help of google translate I also fixed the italian and korean page too.

File: pages.zh/common/apg.md [hunk 17]
```diff
@@ -1,6 +1,7 @@
 # apg
 
 > 生成任意复杂度的随机密码.
+> 更多信息： <https://manpages.debian.org/latest/apg/apg.1.html>.
```
Comment (navarroaxel) [inline pages.zh/common/apg.md:4]:
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apg/apg.1>.
```

File: pages/common/ar.md [hunk 18]
```diff
@@ -1,23 +1,24 @@
 # ar
 
-> Create, modify, and extract from archives (.a, .so, .o).
+> Create, modify, and extract from archives (`.a`, `.so`, `.o`).
+> More information: <https://man.archlinux.org/man/ar.1>.
 
 - Extract all members from an archive:
 
-`ar -x {{libfoo.a}}`
+`ar -x {{path/to/archive.a}}`
```
Comment (navarroaxel) [inline pages/common/ar.md:8]:
```suggestion
`ar -x {{path/to/file.a}}`
```

File: pages/common/ar.md [hunk 19]
```diff
@@ -1,23 +1,24 @@
...
 
 - List the members of an archive:
 
-`ar -t {{libfoo.a}}`
+`ar -t {{path/to/archive.a}}`
```
Comment (navarroaxel) [inline pages/common/ar.md:12]:
```suggestion
`ar -t {{path/to/file.a}}`
```

File: pages/common/ar.md [hunk 20]
```diff
@@ -1,23 +1,24 @@
...
 
 - Replace or add files to an archive:
 
-`ar -r {{libfoo.a}} {{foo.o}} {{bar.o}} {{baz.o}}`
+`ar -r {{path/to/archive.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
```
Comment (navarroaxel) [inline pages/common/ar.md:16]:
```suggestion
`ar -r {{path/to/file.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
```

File: pages/common/ar.md [hunk 21]
```diff
@@ -1,23 +1,24 @@
...
 
 - Insert an object file index (equivalent to using `ranlib`):
 
-`ar -s {{libfoo.a}}`
+`ar -s {{path/to/archive.a}}`
```
Comment (navarroaxel) [inline pages/common/ar.md:20]:
```suggestion
`ar -s {{path/to/file.a}}`
```

File: pages/common/ar.md [hunk 22]
```diff
@@ -1,23 +1,24 @@
...
 
 - Create an archive with files and an accompanying object file index:
 
-`ar -rs {{libfoo.a}} {{foo.o}} {{bar.o}} {{baz.o}}`
+`ar -rs {{path/to/archive.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
```
Comment (navarroaxel) [inline pages/common/ar.md:24]:
```suggestion
`ar -rs {{path/to/file.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages/common/column.md [hunk 23]
```diff
@@ -1,20 +1,21 @@
...
+`printf "header1 header2\nbar foo\n" | column --output-width {{30}}`
 
-- Split columns automatically and auto-align in a tabular format:
+- Split columns automatically and auto-align them in a tabular format:
 
-`printf "header1 header2\nbar foo\n" | column -t`
+`printf "header1 header2\nbar foo\n" | column --table`
 
-- Specify column delimiter character for the -t option (e.g. "," for csv); default is whitespace:
+- Specify the column delimiter character for the `--table` option (e.g. "," for csv) (defauls to whitespace):
```
Comment (bl-ue) [inline pages/common/column.md:15]:
```suggestion
- Specify the column delimiter character for the `--table` option (e.g. "," for csv) (defaults to whitespace):
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Comment by marchersimon** [comment]
Is this ready to be merged @bl-ue?

**Review by sbrl — CHANGES_REQUESTED** [review]
Requesting changes to avoid an accidental merge until we have a consensus on #5143.

**Comment by marchersimon** [comment]
I think the decision with `manned.org` is final, so I'll replace the links.

**Review by Waples — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Pull request merged**